src/pocketmine/lang/locale

```diff
@@ -1 +0,0 @@
-Subproject commit a7d53d9229337323220c7370916a153accc0ebf3
```

Comment:
I'm afraid it's quite hard to revert the submodule changes at this stage (how did you even do it!). Initially, you should clone the repo with the --recursive flag, or execute `git submodule update --init --recursive` after cloning.

At this stage,  https://xkcd.com/1597/ seems to be the easiest method xD